cypress/support/bigtest.js

```diff
@@ -0,0 +1,32 @@
+import { bigtestGlobals } from '@bigtest/globals';
+
+function interact(interaction, runnerState) {
+  bigtestGlobals.runnerState = runnerState;
+
+  return cy.document({ log: false }).then((doc) => {
+    bigtestGlobals.document = doc;
+
+    return interaction;
+  }).then(() => {
+    Cypress.log({
+      displayName: runnerState,
+      message: interaction.description
+    });
+  });
+}
+
+Cypress.Commands.add('do', (interaction) => {
+  if (Array.isArray(interaction)) {
+    interaction.map(i => interact(i, 'step'));
+  } else {
+    interact(interaction, 'step');
+  }
+});
+
+Cypress.Commands.add('expect', (interaction) => {
+  if (Array.isArray(interaction)) {
+    interaction.map(i => interact(i, 'assertion'));
+  } else {
+    interact(interaction, 'assertion');
+  }
+});
```

Comment:
This is the integration we provide in `@bigtest/cypress`

How can we improve that package so that you don't have to copy and paste that code?

Reply:
I think it's better to expose just your commands like other do https://docs.cypress.io/plugins/directory#Custom%20Commands

Reply:
We followed the template of plugins like https://github.com/RcKeller/cypress-redux and https://github.com/component-driven/cypress-axe

So if you add 
```js
import `@bigtest/cypress`;
```

 it should register the `do` and `expect` commands for you.

Reply:
nice, I'll try to avoid this copy paste from original repo, makes sense

Reply:
yep, thanks for pointing  this